File: ToptanTrackRepository/src/main/java/com/ahbap/ToptanTrack/TotalProfitDto.java
package com.ahbap.ToptanTrack;

import lombok.AllArgsConstructor;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.ToString;

import java.math.BigDecimal;

@EqualsAndHashCode
@ToString
@AllArgsConstructor
public class TotalProfitDto {

    public BigDecimal totalProfit;
    public BigDecimal byPrice;
    public BigDecimal sell_Price;
    public String name;
}

File: ProductApp/app/src/main/java/com/ahbap/android/app/productapp/retrofit/dto/TotalProfitDto.kt
package com.ahbap.android.app.productapp.retrofit.dto

import java.math.BigDecimal

data class TotalProfitDto(
    var totalProfit: BigDecimal? = null,
    var byPrice: BigDecimal? = null,
    var sell_Price: BigDecimal? = null,
    var name: String? = null,
    var stock: String? = null)

